### ISSUE TITLE
[Effect Suggestion]

### ISSUE THREAD

**Issue by 810CK-A1D** [gta-chaos-mod/ChaosModV#541]
**Name of effect**
Force switch

**Effect Description**
Forces player to switch another character, if there is a better name, feel free to change it

**Comment by anto4440** [comment]
If you were on a mission or something like that, the effect wouldn't work.

**Comment by SeanMirrsen** [comment]
> If you were on a mission or something like that, the effect wouldn't work.

That's true, however it does make me wonder. Switching to another character is possible in some missions, and like all other times it's basically a keybind. So you could activate the relevant commands, whether or not they work, and if something does happen, then all is well. You can't discourage effects like Walk On Water or Ramp Jam, just because the player isn't usually near any water, and isn't always in a road vehicle.

**Comment by anto4440** [comment]
Yes, i can. 'Things are getting wavy' was disabled due to player never being in water.

**Comment by SeanMirrsen** [comment]
And yet Walk on Water was added. And when it activates at appropriate time, the results are hilarious.
